docs/lexicon.html

```diff
@@ -14,8 +14,9 @@ <h1>The Lexicon of Please</h1>
       <li><a href="#subrepo">Subrepo rules</a></li>
     </ul>
 
-    <h2><a name="builtins">Builtin functions</a></h2>
-
+    <h2><a name="builtins">Built in functions</a></h2>
+    <p>In addition to <a href="/language.html">the build language</a> built in functions, Please also has built
```

Comment:
"built in functions" is doubled, feels awkward.

maybe "Please also has a set of built in rules"?